modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -65,33 +65,32 @@ rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0
 ----
 
 
-** For installations on FCP-type disks, complete the following tasks:
-... Use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed. For multipathing repeat this step for each additional path. 
-... For multipathing, set the following parameter: `rd.multipath=default`.
-... For multipathing, set the install device as: `coreos.inst.install_dev=/dev/mapper/mpatha`.
-... For single-path installation, set the install device as: `coreos.inst.install_dev=sda`.
```

Comment:
These explanations are required in order to have a working multipath setup on Z. On Z enabling multipath is a day one and day two operation. All paths need to be added to the parm line during installation. The enabling of mulitpath is required after installation, as documented below. If this description is removed, multipath does not work on Z.

Reply:
I've restored these day 1/day 2 multipathing to their previous state. The only changes to `modules/installation-ibm-z-user-infra-machines-iso.adoc` are to remove trailing whitespaces.